mage_ai/shared/config.py

```diff
@@ -56,3 +56,26 @@ def load_extra_config(self) -> Dict:
 
     def to_dict(self) -> Dict:
         return asdict(self)
+
+    @classmethod
+    def load_file(self, config_path: Optional[str] = None, config: Optional[Dict] = None) -> Dict:
+        """
+        Load configuration from a file and merge it with the provided config.
+
+        Parameters:
+            config_path: Path to the configuration file.
+            config: Existing configuration to merge with.
+
+        Returns:
+            Merged dictionary of configurations.
+        """
```

Comment:
You can add `config = config or {}` at the beginning so that you don't need to duplicate the code of `config or {}`